pandas/core/arrays/datetimelike.py

```diff
@@ -1665,12 +1665,7 @@ def _groupby_op(
                 raise TypeError(f"datetime64 type does not support {how} operations")
             if how in ["any", "all"]:
```

Comment:
Could you combine this `if` with the one above?

Reply:
done